### ISSUE TITLE
Tests Hang Using Visual Studio 2015

### ISSUE THREAD

**Issue by martincostello** [martincostello/sqllocaldb#8]
Found while investigating issue #7.

When running the tests in Visual Studio 2015 on a Windows 8.1 machine with Visual Studio 2013 also installed, the tests hang at the end of the test run.

Issue not observed on a machine with only Windows 10 and Visual Studio 2015 installed, or a machine running Windows 7 with Visual Studio 2013 and 2015.

**Comment by martincostello** [comment]
Test classes that contain tests that cause the hang include:

 1. ```SqlLocalDbApiTests```
 1. ```SqlLocalDbInstanceTests```
 1. ``TemporarySqlLocalDbInstanceTests````

**Comment by martincostello** [comment]
The hang no longer occurs if ```SqlLocalDb.runsettings``` is not used and code coverage is disabled. Likely a different side effect of the same underlying issue as #7.

**Comment by martincostello** [comment]
Appears to be a duplicate of #7, just with a different manifestation. Hang has been unblocked for now by disabling code coverage and not specifying a settings file to use with the Visual Studio Test Runner.

**Comment by martincostello** [comment]
Turns out this wasn't a duplicate of #7, but was caused by a change in behaviour in AppVeyor CI that caused Windows to produce a UI pop-up that couldn't be dismissed in an attempt to load an invalid DLL was made.

Fixed by 4916754572080f9af60a5f2ef7454186a36c1d1e.
